lib/vsc/install/testing.py

```diff
@@ -142,7 +142,7 @@ def assertErrorRegex(self, error, regex, call, *args, **kwargs):
         try:
             call(*args, **kwargs)
             str_kwargs = ['='.join([k, str(v)]) for (k, v) in kwargs.items()]
-            str_args = ', '.join(map(str, args) + str_kwargs)
+            str_args = ', '.join([str(x) for x in args] + str_kwargs)
```

Comment:
i'd prefer the py23 module with proper funtion for this.

Reply:
As discussed in Slack, I'm not in favor of adding a module to `vsc-install` which results in making it a runtime dependency, doesn't make sense imho...

If so, we might as well merge `vsc-install` and `vsc-base` into one huge library (and maybe collapse `vsc-utils` into it while we're at it).

Generally speaking, using `map` isn't recommend anyway I think, especially not where having an iterator doesn't make much sense (like here).

Reply:
again, the runtime dep is only temporary, until we drop py2 (ie probably in 2021). merging vsc-utils is a no-go due to its dependencies